framework/src/main/java/org/checkerframework/framework/type/typeannotator/IrrelevantTypeAnnotator.java

```diff
@@ -1,56 +1,83 @@
 package org.checkerframework.framework.type.typeannotator;
 
-import java.util.Set;
-import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.type.TypeMirror;
 import org.checkerframework.framework.qual.RelevantJavaTypes;
 import org.checkerframework.framework.type.AnnotatedTypeMirror;
 import org.checkerframework.framework.type.GenericAnnotatedTypeFactory;
+import org.checkerframework.javacutil.BugInCF;
 
 /**
  * Adds annotations to types that are not relevant specified by the {@link RelevantJavaTypes} on a
  * checker.
  */
 public class IrrelevantTypeAnnotator extends TypeAnnotator {
 
-  /** Annotations to add. */
-  private final Set<? extends AnnotationMirror> annotations;
-
   /**
-   * Annotate every type with the annotationMirror except for those whose underlying Java type is
-   * one of (or a subtype of) a class in relevantClasses. (Only adds annotationMirror if no
-   * annotation in the hierarchy are already on the type.) If relevantClasses includes
-   * Object[].class, then all arrays are considered relevant.
+   * Annotate every type except for those whose underlying Java type is one of (or a subtype or
+   * supertype of) a class in relevantClasses. (Only adds annotationMirror if no annotation in the
+   * hierarchy are already on the type.) If relevantClasses includes Object[].class, then all arrays
+   * are considered relevant.
    *
-   * @param typeFactory AnnotatedTypeFactory
-   * @param annotations annotations to add
+   * @param atypeFactory a GenericAnnotatedTypeFactory
    */
   @SuppressWarnings("rawtypes")
-  public IrrelevantTypeAnnotator(
-      GenericAnnotatedTypeFactory typeFactory, Set<? extends AnnotationMirror> annotations) {
-    super(typeFactory);
-    this.annotations = annotations;
+  public IrrelevantTypeAnnotator(GenericAnnotatedTypeFactory atypeFactory) {
+    super(atypeFactory);
   }
 
   @Override
   protected Void scan(AnnotatedTypeMirror type, Void aVoid) {
-    switch (type.getKind()) {
+    // TODO: This variable name will change to `atypeFactory`, after #5919.
```

Comment:
Do you want to make that change before merging this?

Reply:
No.  That renaming is coupled to many other changes in #5919.